$NetBSD: patch-ab,v 1.1 2009/03/14 11:09:37 apb Exp $

Changes from NetBSD xsrc/external/mit/xf86-input-mouse/dist/\
src/mouse.c

----------------------------
revision 1.2
date: 2009/02/02 03:05:35;  author: christos;  state: Exp;  lines: +4 -0
Call a SetupMouse function to condition the fd.
----------------------------

--- src/mouse.c.orig	2008-03-20 20:55:51.000000000 +0200
+++ src/mouse.c
@@ -2538,6 +2538,10 @@ SetupMouse(InputInfoPtr pInfo)
     Bool automatic = FALSE;
 
     pMse = pInfo->private;
+
+    if (osInfo->SetupMouse)
+	if (!osInfo->SetupMouse(pInfo))
+	    return FALSE;
     
     /* Handle the "Auto" protocol. */
     if (pMse->protocolID == PROT_AUTO) {
